NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift

```diff
@@ -21,9 +21,9 @@ class NVActivityIndicatorAnimationSquareSpin: NVActivityIndicatorAnimationDelega
         animation.timingFunctions = [timingFunction, timingFunction, timingFunction, timingFunction]
         animation.values = [
             NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: 0), createRotateYTransform(angle: 0))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(M_PI)), createRotateYTransform(angle: 0))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(M_PI)), createRotateYTransform(angle: CGFloat(M_PI)))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: 0), createRotateYTransform(angle: CGFloat(M_PI)))),
+            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(Double.pi)), createRotateYTransform(angle: 0))),
```

Comment:
I made the changes using replace all, the old pi constants were all doubles and I didn't realize that in Swift 3 there were now non-double constants for pi available as well and that it meant I could optimize some casting that wouldn't have been possible before. So, no good reason, just a bit of ignorance.

I'll push a commit with the redundant casting stripped out.